skimage/restoration/_denoise.py

```diff
@@ -258,7 +258,7 @@ def denoise_bilateral(image, win_size=None, sigma_color=None, sigma_spatial=1,
 @utils.channel_as_last_axis()
 @utils.deprecate_multichannel_kwarg()
 @utils.deprecate_kwarg({'max_iter': 'max_num_iter'}, removed_version="1.0")
-def denoise_tv_bregman(image, weight, max_num_iter=100, eps=1e-3,
+def denoise_tv_bregman(image, weight=1.0, max_num_iter=100, eps=1e-3,
```

Comment:
I think it would be reasonable to choose a weight here that gives a similar visual appearance to the default weight=0.1 used by denoise_tv_chambolle. Perhaps a value closer to 5.0 is a better default for images in floating point range [0.0, 1.0]. See for example the figure below adapted from an existing example that shows that `weight=1.0` is pretty smoothed. As an aside, the cat image used here is not an ideal candidate for TV-based denoising. It is most effective for piecewise constant (i.e. cartoon-like) images.

![bregman_defaults](https://user-images.githubusercontent.com/6528957/126913729-7ce62ee6-2c45-41d2-ae59-ac457c371969.png)

Reply:
It is not something to address in this PR, but it may confuse users that "weight" being larger -> more smoothed for `denoise_tv_chambolle`, but less smoothed for `denoise_tv_bregman`. This has to do with a difference in how the equations are written in the two publications (weight in numerator vs. denominator)

Reply:
Sorry for the misunderstanding, and thanks for the prompt response. I fixed it to 5.0 and waiting for the tests to pass!  Risako